src/components/toast/toast.tsx

```diff
@@ -1,5 +1,8 @@
-import { createContext, ReactNode, useContext, useState } from 'react';
+import React, { createContext, ReactNode, useContext, useState } from 'react';
 import styled from 'styled-components';
+import { fontSize, pxToRem, spacing } from '../../styles';
+import { useStore } from '../../helpers';
+import { IconButton } from '../iconButton/iconButton';
 
 const ToastContext = createContext(undefined);
```

Comment:
changed `undefined` to `null`

Reply:
It was your code, so i didn't change it while working on my part of code (just informing you).